### ISSUE TITLE
Spanish (ES) version of the tutorial

### ISSUE THREAD

**Issue by cdelaorden** [evancz/elm-architecture-tutorial#71]
Hi, I saw the Japanese version of the README and just translated the whole tutorial to Spanish, in case you're interested in adding it as well.

It's right now in [my fork](https://github.com/cdelaorden/elm-architecture-tutorial/blob/dev/README.es.md), but I can send you a PR if you prefer that way.

Great tutorial by the way, to all people involved. :)

**Comment by pirj** [comment]
I think it's always better to send a PR.

**Comment by evancz** [comment]
Sorry I did not reply earlier! I did not know how to handle translations at all at that time, and I wasn't sure what to do.

We still do not have an official process for translations, but the folks working on Japanese translations are active in the #guide-translation channel on Slack. I believe they have some tools and processes to help things go more smoothly. I think talking with folks there is the best way to try to coordinate on getting more translations available.

I am open to having a more official process, but I don't really know enough to plan it out myself.
